lib/elixir/lib/map.ex

```diff
@@ -27,8 +30,48 @@ defmodule Map do
   @doc """
   Returns a new empty map.
   """
+  @spec new :: map
   def new, do: %{}
 
+  @doc """
+  Creates a `Map` from an enumerable.
```

Comment:
`Creates a map from an enumerable.` `Map` in this context is a module reference.